lms/djangoapps/mobile_api/v1/users/serializers.py

```diff
@@ -0,0 +1,26 @@
+"""
+Serializer for user v1 API
+"""
+
+from rest_framework import serializers
+from mobile_api.users.serializers import CourseEnrollmentSerializer as CourseEnrollmentSerializerBase
+from openedx.features.course_duration_limits.access import get_user_course_expiration_date
+from student.models import CourseEnrollment
+
+
+class CourseEnrollmentSerializer(CourseEnrollmentSerializerBase):
+    """
+    Extend base CourseEnrollmentSerializer, add expiration field
+    """
+    expiration = serializers.SerializerMethodField()
+
+    def get_expiration(self, model):
+        """
+        Returns expiration date for a course, if any or null
+        """
+        return get_user_course_expiration_date(model.user, model.course)
+
+    class Meta(object):
+        model = CourseEnrollment
+        fields = ('expiration', 'created', 'mode', 'is_active', 'course', 'certificate')
```

Comment:
in the unversioned serializers.py course is course = CourseOverviewField(source="course_overview", read_only=True).  
What is course here?

Reply:
it is the same. course field gets inherited from CourseEnrollmentSerializerBase